tests/test-verify-pool-volumes.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: verify the volumes
+  include_tasks: "test-verify-volume.yml"
+  loop: "{{ storage_test_pool.volumes }}"
+  loop_control:
+    loop_var: storage_test_volume
+  when: storage_test_pool.volumes | length > 0
```

Comment:
Is this `when:` needed? Aren't loops over zero-length lists just skipped?

Reply:
Yeah, shouldn't be needed.

Reply:
If it is possible that `storage_test_pool.volumes` is undefined, then perhaps something like
`when: storage_test_pool.volumes | d([])`
or even omit the `when` clause and use 
`loop: "{{ storage_test_pool.volumes | d([]) }}"`